daily-challenges/2024-12/18/problem.md

```
@@ -0,0 +1,65 @@
+  }
+
+  interface CongThuc {
+    ten: string;
+    nguyenLieu: NguyenLieu[];
+    cachLam: string[];
+    doKho: 1 | 2 | 3;  // độ khó pha chế
+  }
+  ```
+
+- Viết class `QuanLyDoUong` với các phương thức:
+  + `kiemTraKhaNangPhaChe(congThuc: CongThuc)`: Kiểm tra có thể pha được không
```

Markdownlint (0.37.0)

Expected: dash; Actual: plus
Unordered list style

(MD004, ul-style)